src/System.Private.CoreLib/src/System/Threading/ClrThreadPool.cs

```diff
@@ -46,6 +46,8 @@ private struct CacheLineSeparated
         private static int s_completionCount = 0;
         private static int s_threadAdjustmentInterval;
 
+        private static volatile int s_numRequestedWorkers = 0;
+
         static ClrThreadPool()
```

Comment:
Explicit static constructors are not good for performance - they will cause each static method prolog to have cctor trigger. It is better to do the initialization inline.